### PR TITLE
Fix getUsers to return UserStats

### LINKED ISSUE #1339: Fix getUsers to return UserStats
**Actual Behaviour**
Currently it shows a 500 Failed to fetch the requested list error.

**Expected Behaviour**
Fix it.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #1339 
Screenshots for the change: 
Screenshot 2019-07-27 at 10 40 26 PM

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@akshatnitd @fragm3 Please review.

**Comment by harsh-jindal** [comment]
@fragm3 You are referring to this right https://github.com/fossasia/susi_server/commit/4940e25810ebdc929e99440a0794b2ebc2d21bcd
The error was due to `Objects.requireNonNull` function I have removed that

File: src/ai/susi/server/api/aaa/GetUsers.java [hunk 1]
```diff
@@ -148,7 +148,7 @@ public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Aut
                   }
 
                   if(accounting.getJSON().has("signupTime")) {
-                    String signupTime = accounting.getJSON().getString("signupTime").substring(0, 8);
+                    String signupTime = accounting.getJSON().getString("signupTime").substring(8,16);
```
Comment (fragm3) [inline src/ai/susi/server/api/aaa/GetUsers.java]: This is different when you run locally and on actually instantiated server. Undo this

Comment (harsh-jindal) [reply src/ai/susi/server/api/aaa/GetUsers.java]: @fragm3 Done

**Pull request merged**